### ISSUE TITLE
Save_mic_level error when turning privacy mode on

### ISSUE THREAD

**Issue by maddoxjny** [briis/unifiprotect#409]
### Home Assistant Version?

2021.12.2

### UniFi Protect Version?

Latest

### UniFi Protect HA Integration version?

Latest Stable

### What happened?

when turning privacy mode on  receive the error below this happens in a automation and when I enable it manually .   I then go into each camera set the microphone level to anything above 0 and set my recording to always.  Then when I toggle the privacy mode it works as expected.  Unfortunately this issue keeps coming back.   Pretty sure its on reboot.    

### Relevant log output

```shell
Service - anyone home?: Choose at step 1: choice 2: Error executing script. Unexpected error for call_service at pos 2: 'save_mic_level'
Service - anyone home?: Error executing script. Unexpected error for choose at pos 1: 'save_mic_level'
While executing automation automation.disable_inside_cameras_when_someone_is_home
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 381, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 584, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1495, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1530, in _execute_service
    await handler.job.target(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 209, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 663, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 896, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 700, in _handle_entity_call
    await result
  File "/config/custom_components/unifiprotect/switch.py", line 177, in async_turn_off
    self._device_data["save_mic_level"],
KeyError: 'save_mic_level'
```

**Comment by AngellusMortis** [comment]
Please add your UniFi Protect and HA Integration versions. "Latest" is not very helpful.

If you are on 0.10.x, please upgrade to 0.11-beta first to see if you still have the issue. However, please not that many of the services were deprecated in 0.11 and the service you are using is likely deprecated.

**Comment by AngellusMortis** [comment]
We released 0.11 to stable so I am going to close this one. Please update to 0.11 if you can and if you are still having the issue, let us know.
